public/locales/en/translation.json

```diff
@@ -1,5 +1,35 @@
 {
-  "main": {
-    "appBarHeader": "Dashboard"
+  "categories": {
+    "systems": "Systems",
+    "trees": "Fault trees",
+    "worksheets": "FMEA worksheets",
+    "tables": "FHA tables"
+  },
+  "create": {
+    "system": "New system",
```

Comment:
@blcham @LaChope We should discuss about capital and small letters in English (consistent style CS vs. EN).

Reply:
IMO if we want it consistent across CS and EN, we should have the first word with capital letter and other words with small letters (so that we have the same in EN and CS)

Reply:
I leave it like this. "New system". Only first letter is capital